.eslintrc.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
     "no-useless-constructor": "off",
     "space-before-function-paren": "off",
     "semi": "off",
+    "@typescript-eslint/no-inferrable-types": "off",
```

Comment:
This rules conflicts with @typescript-eslint/typedef, when you use clases, example:
```
export class UserModel1574858958271 implements MigrationInterface {
  name = 'safasfsf'; // expected name to have a type annotation eslint(@typescript-eslint/typedef)
  name: string = 'dafasfdas'; // Type string trivially inferred from a string literal, remove type annotation.eslint(@typescript-eslint/no-inferrable-types)
... other things
```
Seems more important typedef rule, i turned off because is something that would happen each time with a auto-generated migration